app/post_metadata.go

```diff
@@ -250,7 +250,7 @@ func (a *App) getImagesForPost(post *model.Post, imageURLs []string, isNewPost b
 }
 
 func getEmojiNamesForString(s string) []string {
-	names := model.EMOJI_PATTERN.FindAllString(s, -1)
```

Comment:
Instead of switching this to ALL_EMOJI_PATTERN, could you just get rid of ALL_EMOJI_PATTERN and make EMOJI_PATTERN include the plus sign? We don't need to keep both of them around any more, and we can also remove [this TODO comment](https://github.com/mattermost/mattermost-server/blob/master/model/emoji.go#L24) at the same time